modules/rosa-sts-creating-a-cluster-quickly-ocm.adoc

```diff
@@ -0,0 +1,276 @@
+// Module included in the following assemblies:
+//
+// * rosa_getting_started/rosa-getting-started.adoc
+
+:_content-type: PROCEDURE
+[id="rosa-sts-creating-cluster-using-defaults-ocm_{context}"]
+= Creating a cluster with the default options through {cluster-manager}
+
+Through {cluster-manager-first}, you can use the default options to quickly create a {product-title} (ROSA) cluster that uses the AWS Security Token Service (STS).
+
+You can use `auto` mode to immediately create the required AWS Identity and Access Management (IAM) resources using the current AWS account. `auto` mode is used in the following procedure to immediately create the account-wide IAM roles and policies, including the Operator policies, as well as the OpenID Connect (OIDC) identity provider.
+
+.Prerequisites
+
+* You have completed the AWS prerequisites for ROSA with STS.
+* You have available AWS service quotas.
+* You have enabled the ROSA service in the AWS Console.
+* You installed and configured the latest AWS (`aws`), ROSA (`rosa`), and OpenShift (`oc`) CLIs on your workstation.
+* You logged in to your Red Hat account by using the `rosa` CLI.
+* You verified that the AWS Elastic Load Balancing (ELB) service role exists in your AWS account.
++
+[NOTE]
```

Comment:
Would it make more sense to add this note under the bullet that talks about using the CLI?

Reply:
Great suggestion, thanks. I have resolved this throughout the PR.